sdk/extensions/Azure.Extensions.AspNetCore.Configuration.Secrets/README.md

```diff
@@ -15,7 +15,7 @@ dotnet add package Azure.Extensions.AspNetCore.Configuration.Secrets -v 1.0.0-pr
 ### Prerequisites
 
 You need an [Azure subscription][azure_sub] and
-[Azure KeyVault][keyvault_doc] to use this package.
+[Azure Key Vault][keyvault_doc] to use this package.
 
 To create a new KeyVault, you can use the [Azure Portal][keyvault_create_portal],
```

Comment:
More here. I'd just do a global search and replace, though watch out for links. Maybe do a case-sensitive search for KeyVault -> Key Vault.

Reply:
I thought that's what I did, I'll double check.